src/hooks/useZarrMetadata.ts

```diff
@@ -77,70 +145,43 @@ export default function useZarrMetadata() {
-          file => file.name === '.zarray'
-        );
+
+        const zarrayFile = await getFile('.zarray');
         if (zarrayFile) {
-          try {
-            try {
-              const arr = await getZarrArray(imageUrl);
-              if (cancelRef.cancel) {
-                return;
-              }
-              const shapes = [arr.shape];
-              setMetadata({
-                arr,
-                shapes,
-                multiscale: undefined,
-                omero: undefined,
-                scales: undefined,
-                zarrVersion: 2
-              });
-            } catch (error) {
-              log.error('Error fetching Zarr array:', error);
-              if (cancelRef.cancel) {
-                return;
-              }
-              setThumbnailError('Error fetching Zarr array');
-            }
-          } catch (error) {
-            log.error('Error fetching Zarr array metadata:', error);
-          }
+          checkZarrArray(imageUrl, 2, cancelRef);
         } else {
-          const zattrsFile = fileBrowserState.files.find(
-            file => file.name === '.zattrs'
-          );
+          const zattrsFile = await getFile('.zattrs');
           if (zattrsFile) {
-            try {
-              const zattrs = (await fetchFileAsJson(
+            const attrs = (await fetchFileAsJson(
+              fileBrowserState.currentFileSharePath.name,
+              zattrsFile.path,
+              cookies
+            )) as any;
+            if (attrs.multiscales) {
+              checkOmeZarrMetadata(imageUrl, 2, cancelRef);
+            }
+          } else {
+            const zarrJsonFile = await getFile('zarr.json');
+            if (zarrJsonFile) {
+              const attrs = (await fetchFileAsJson(
                 fileBrowserState.currentFileSharePath.name,
-                zattrsFile.path,
+                zarrJsonFile.path,
                 cookies
               )) as any;
-              if (zattrs.multiscales) {
-                setThumbnailError(null);
-                try {
-                  setOmeZarrUrl(imageUrl);
-                  const metadata = await getOmeZarrMetadata(imageUrl);
-                  if (cancelRef.cancel) {
-                    return;
-                  }
-                  setMetadata(metadata);
-                  setLoadingThumbnail(true);
-                } catch (error) {
-                  log.error(
-                    'Exception fetching OME-Zarr metadata:',
-                    imageUrl,
-                    error
-                  );
-                  if (cancelRef.cancel) {
-                    return;
-                  }
-                  setThumbnailError('Error fetching OME-Zarr metadata');
+              if (attrs.node_type === 'array') {
+                checkZarrArray(imageUrl, 3, cancelRef);
+              } else if (attrs.node_type === 'group') {
+                if (
+                  attrs.attributes?.multiscales ||
```

Comment:
While Neuroglancer accepts `multiscales` outside of `ome` in zarr.json, I really think we should at least issue a warning here as we are clearly outside of the OME-NGFF specification when this occurs.